MANIFEST

```diff
@@ -0,0 +1,11 @@
+# file GENERATED by distutils, do NOT edit
```

Comment:
Is this an autogen file? If yes, please do not check it in.

Reply:
yes, it will be generated during source distribution of package for pypi staging. Fine, I will add it to .gitignore